test/game_team_collection_test.rb

```diff
@@ -17,4 +18,13 @@ def test_it_can_return_an_array_all_game_teams
     assert_equal 60, @@game_team_collection.all_game_teams.size
     assert @@game_team_collection.all_game_teams.all? { |game_team| game_team.class == GameTeam}
   end
+
+  def test_it_can_add_season_id
+    @@game_path = './data/game_teams_fixture.csv'
+    @@game_collection = GameCollection.new(@@game_path)
```

Comment:
These two variables don't need to be global. They could just be local to that test method. Honestly not sure what the rule is for making global variables - freely or strictly as needed?

Reply:
Ah yes! I've changed these to local. My understanding of global variables is to use them sparingly since they don't really follow the principle of encapsulation? After converting these local, I confirmed that all tests still pass.

Reply:
I've heard to use them somewhat sparingly, because it can make debugging a bit difficult. But other than that, I'm not entirely sure what the rules are around using them :)